src/vcModals.cpp

```diff
@@ -54,30 +54,29 @@ void vcModals_DrawLoggedOut(vcState *pProgramState)
 }
 
 // Presents user with a message if the specified file exists, then returns false if user declines to overwrite the file
-bool vcModals_OverwriteExistingFile(vcState *pProgramState, const char *pFilename)
+bool vcModals_OverwriteExistingFile(vcState* pProgramState, const char* pFilename, const char* pFileExistingMsg)
 {
   bool result = true;
-  const char *pFileExistsMsg = nullptr;
   if (udFileExists(pFilename) == udR_Success)
   {
     const SDL_MessageBoxButtonData buttons[] = {
       { SDL_MESSAGEBOX_BUTTON_ESCAPEKEY_DEFAULT, 0, vcString::Get("popupConfirmNo") },
       { SDL_MESSAGEBOX_BUTTON_RETURNKEY_DEFAULT, 1, vcString::Get("popupConfirmYes") },
     };
-    pFileExistsMsg = vcStringFormat(vcString::Get("convertFileExistsMessage"), pFilename);
+    if (pFileExistingMsg == nullptr)
+      pFileExistingMsg = vcStringFormat(vcString::Get("convertFileExistsMessage"), pFilename);
```

Comment:
Isn't this a generic message?

Reply:
it is, if there is no specific string given.

Reply:
Sorry, I was not clear at all here, must have been distracted when I wrote it.

What I should have written:
> If this is a generic message, is there any need to have an optional message passed in?

Reply:
As discussed on Teams, this is necessary since vcFlythrough is using a file to indicate that a folder is going to be overwritten rather than a file. Resolving discussion.